lib/tasks/js_rails_routes.rake

```diff
@@ -7,6 +7,7 @@ namespace :js do
     generator.include_names = Regexp.new(ENV['include_names']) if ENV['include_names']
     generator.exclude_names = Regexp.new(ENV['exclude_names']) if ENV['exclude_names']
     generator.output_dir = ENV['output_dir'] if ENV['output_dir']
+    generator.camelize = ENV['camelize'] if ENV['camelize']
```

Comment:
`ENV['camelize']` is a string, not symbol. And if it is an empty string, it should become `nil` to output snake_case.

Reply:
Exactly, thanks.